klog.go

```diff
@@ -998,7 +1010,7 @@ func (l *loggingT) output(s severity, log *logr.Logger, buf *buffer, depth int,
 		}
 		l.mu.Unlock()
 		timeoutFlush(10 * time.Second)
-		os.Exit(255) // C++ uses -1, which is silly because it's anded with 255 anyway.
```

Comment:
This change is wrong. The comments is referring to the operator and.

Reply:
Got it, updated with operator &.